konduit-serving-build/pom.xml

```diff
@@ -210,7 +210,7 @@
                         </configuration>
                     </execution>
                 </executions>
-            </plugin>
+            </plugin>-->
```

Comment:
We should remove the maven shade plugin entirely from this pom. If users need to call the build tool CLI they can do it from the konduit-serving-cli uberjar instead.
(And yes, this commenting-out was my change, I probably should have removed instead of commenting :))